### DISCUSSION TITLE
Add "Total Participation" to daily challenge tooltip

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#29592] [Ideas] [answered]
On the profile, the daily challenge stat displayed is actually the total days where someone participated in the daily challenge. 
I'm proposing including this stat in the tooltip as pictured below, in an effort to remove any confusion.

![Daily Challenge tooltip](https://github.com/user-attachments/assets/3fc5779d-e5a3-4eb7-9cf6-d2c2da25218d)
(I just realized I should probably have posted this in osu-web but oh well, this is implemented in the client too)

**Comment by peppy** [comment, marked as answer]
I can agree with this.

**Reply by Joehuu** [reply]
Web PR: https://github.com/ppy/osu-web/pull/11457

**Reply by vatei** [reply]
> Web PR: https://github.com/ppy/osu-web/pull/11457

Hmmm, it's still not very clear that the "total participation" statistic is referring to the number displayed on the profile
Since it's the main stat I would expect it to be the first one in the tooltip, with the two other important ones

**Reply by peppy** [reply]
I touched on this in the implementation. cc @nanaya how weird does it look if we show it big?

**Reply by nanaya** [reply]
![](https://s.kohaku.love/2024/09/firefox_2024-09-02_14-23-58.png)

**Reply by peppy** [reply]
Maybe okay?

**Reply by vatei** [reply]
Looks normal to me 👍

**Reply by peppy** [reply]
Has been added on osu-web, we'll need to also add it in the client (good-first-issue for sure).

**Reply by Joehuu** [reply]
Made an issue for more visibility for https://github.com/ppy/osu/labels/good%20first%20issue: https://github.com/ppy/osu/issues/29706
